spyder_kernels/customize/code_runner.py

```diff
@@ -30,18 +33,20 @@
 )
 from IPython.core.magic import (
     needs_local_scope,
+    no_var_expand,
+    line_cell_magic,
     magics_class,
     Magics,
     line_magic,
 )
 from IPython.core import magic_arguments
 
 # Local imports
-from spyder_kernels.comms.frontendcomm import frontend_request
+from spyder_kernels.comms.frontendcomm import frontend_request, CommError
 from spyder_kernels.customize.namespace_manager import NamespaceManager
 from spyder_kernels.customize.spyderpdb import SpyderPdb
 from spyder_kernels.customize.umr import UserModuleReloader
-from spyder_kernels.customize.utils import capture_last_Expr, canonic
+from spyder_kernels.customize.utils import capture_last_Expr, canonic, create_pathlist
```

Comment:
```suggestion
from spyder_kernels.customize.utils import (
    capture_last_Expr, canonic, create_pathlist)
```

The line is too long.